### PR TITLE
Order of command line flags

### PR DESCRIPTION
**Pull request by nellicus**
specifying the order required for cli -Des notation arguments

### CODE REVIEW

**Comment by nik9000** [comment]
Relates to e27ede48ce6f275f80e8b59bc6dc7ad2849c64fc and #13737.

**Comment by nik9000** [comment]
Looks good to me. Now that I look at it I wonder why you have to specify the parameters in that order.

**Comment by nellicus** [comment]
@nik9000 I am not sure why but it caused my existing config to fail

File: docs/reference/migration/migrate_2_0/settings.asciidoc [hunk 1]
```diff
@@ -1,5 +1,24 @@
 === Setting changes
 
+==== Command line flags
+
+Command line flags using notation `-Des.*` must be now specified as the first arguments.
```
Comment (nik9000) [inline docs/reference/migration/migrate_2_0/settings.asciidoc]: Its actually anything that starts with a single dash I believe. Which is mostly `-D`.

The rest of the change is great.

**Pull request merged**